http/src/main/resources/reference.conf

```diff
@@ -442,6 +442,14 @@ contentSecurityPolicy {
   ]
 }
 
+refererConfig {
+  # list of valid hosts is environment specific so it's overriden in firecloud develop
+  # add dev valid hosts here
+  validHosts = [
+    "none"
```

Comment:
As mentioned in 1-1, can probably put dev hosts here instead of "none"

Reply:
will update!